test/parse-result.js

```diff
@@ -0,0 +1,135 @@
+/*
+ * Tests for parse result namespace elements, including all their
+ * convenience properties and methods.
+ */
+
+import chai, {Assertion, expect} from 'chai';
+
+import minim from 'minim';
+import minimApiDescription from 'minim-api-description';
+import minimParseResult from '../src/parse-result';
+
+const namespace = minim.namespace()
+  .use(minimApiDescription)
+  .use(minimParseResult);
```

Comment:
Is the API Description namespace a requirement to make the Parse Result namespace work? If so, do you think it would be appropriate to load the API Description namespace while loading the Parse Result one?

If not, please ignore :)